src/sentry/remote_config/docs/api.md

```diff
@@ -6,40 +6,30 @@ Host: https://sentry.io/api/0
 
 @cmanallen
 
-## Configuration [/projects/<organization_id_or_slug>/<project_id_or_slug>/configuration/]
-
-### Get Configuration [GET]
-
-Retrieve the project's configuration.
+## Configuration Proxy [/remote-config/projects/<project_id>/]
 
-**Attributes**
+Temporary configuration proxy resource.
 
-| Column   | Type           | Description                                   |
-| -------- | -------------- | --------------------------------------------- |
-| features | array[Feature] | Custom, user-defined configuration container. |
-| options  | Option         | Sentry SDK options container.                 |
+- Parameters
+  - environment (optional, string) - The environment to scope the configuration by.
```

Comment:
if this is optional, then the `env=null` config becomes important.

if it's not optional, and the env is missing from the database are we falling back to the `null` config or nothing?

Reply:
Right now I'm not sure what to do.  Environment is optionally defined so it is optional.  We can have a null config or fallback to a default like "production".  I think falling back to "production" is best but I'm not totally sure.  lmk your thoughts.